### DISCUSSION TITLE
有没有查询单表占用空间大小的方法？

### DISCUSSION THREAD

**Discussion by Eric-xiaxl** [taosdata/TDengine#4755] [General]
1.mysql采用下面方法查询：
`select DATA_LENGTH
 from INFORMATION_SCHEMA.tables
 where table_schema = '数据库名' and table_name = '表名';`

2.oracle采用下面方法查询：
`SELECT segment_name AS TABLENAME,
       BYTES B,
       BYTES / 1024 KB,
       BYTES / 1024 / 1024 MB
  FROM user_segments
where segment_name = upper('表名');`

但是TDEngine能有什么办法查询单表占用空间大小，我找不到，求帮助。

**Comment by sangshuduo** [comment]
暂时没有。TDengine根据 keep 时间自动管理数据存储，这个功能不是特别需要。
